AK/RedBlackTree.h

```diff
@@ -131,6 +131,25 @@ class BaseRedBlackTree {
         return candidate;
     }
 
+    static Node* find_smallest_not_below(Node* node, K key)
+    {
+        while (true) {
+            if (!node)
+                return node;
```

Comment:
wouldn't this be nicer as just:
```suggestion
        while (node) {
```
and then a `return node` at the end like `find_largest_not_above`?

Reply:
Sure, I was just writing what came out of my brain :P